src/pages/kyunghyunKim/Login/LoginKyung.scss

```
@@ -0,0 +1,68 @@
+* {
+  box-sizing: border-box;
+}
+
+html,
+body {
+  width: 100%;
+  height: 100%;
+}
+
+.westagram {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.westaContainer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 80px 80px 200px 80px;
+  border: 1px solid rgb(233, 230, 230);
+}
+
+.logo {
+  font-family: 'Lobster', cursive;
+  font-size: 50px;
+}
+
+.login {
+  margin-top: 50px;
+  width: 295px;
+}
```

```suggestion
.westagram {
  display: flex;
  justify-content: center;
  align-items: center;
  height: 100vh;
  
  .westaContainer {
  display: flex;
  flex-direction: column;
  align-items: center;
  padding: 80px 80px 200px 80px;
  border: 1px solid rgb(233, 230, 230);

    .logo {
    font-family: 'Lobster', cursive;
    font-size: 50px;
    }

    .login {
      margin-top: 50px;
      width: 295px;
    }
  }
}
```

sass 의 nesting 기능 활용해 주세요